src/res_comments_table.ml

```diff
@@ -1310,9 +1311,36 @@ and walkExpression expr t comments =
         walkExpression callExpr t inside;
         after)
     in
-    let afterExpr, rest = partitionAdjacentTrailing callExpr.pexp_loc after in
-    attach t.trailing callExpr.pexp_loc afterExpr;
-    walkList (arguments |> List.map (fun (_, e) -> ExprArgument e)) t rest
+    if ParsetreeViewer.isJsxExpression expr then (
+      let props =
+        arguments
+        |> List.filter (fun (label, _) ->
+               match label with
+               | Asttypes.Labelled "children" -> false
+               | Asttypes.Nolabel -> false
+               | _ -> true)
+      in
+      let maybeChildren =
+        arguments
+        |> List.find_opt (fun (label, _) ->
+               label = Asttypes.Labelled "children")
+      in
+      match maybeChildren with
+      | None -> ()
+      | Some (_, children) ->
+        let leading, inside, _ = partitionByLoc after children.pexp_loc in
+        if props = [] then
+          let afterExpr, _ =
```

Comment:
What happen to the second argument, `_` in this case?
Why can it be ignored?

Reply:
Trailing comments is always an empty list, so there is no need to deal with it.

Reply:
How about adding a comment saying that.